app/views/layouts/design_system.html.erb

```diff
@@ -28,7 +28,11 @@
 
   <div class="legacy-whitehall">
     <div class="environment-<%= environment %> navbar-wrapper">
-      <%= render partial: "shared/header" %>
+      <% if show_new_header? %>
+        <%= render partial: "shared/header" %>
+      <% else %>
+        <%= render partial: "shared/legacy_header" %>
+      <% end %>
     </div>
   </div>
```

Comment:
I think this will need to be restructured a bit like below so that when using the new header we don't nest it in the classes we don't need (`legacy-whitehall`, `navbar-wrapper` etc.)

```
<% if show_new_header? %>
    <%= render partial: "shared/header" %>
<% else %>
  <div class="legacy-whitehall">
    <div class="environment-<%= environment %> navbar-wrapper">
        <%= render partial: "shared/legacy_header" %>
    </div>
  </div>
<% end %>
```